package.json

```diff
@@ -3,7 +3,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "react-scripts start",
+    "start": "npm i && react-scripts start",
```

Comment:
Create an issue to modify the README so no one is installing twice? Unless it doesn't affect anything of course.

Reply:
I don't think I ever mentioned to do an npm i every time you pull down a new branch to review. I just stuck it on here so no one ever needs to remember (also a standard practice) good looking out, though. I may need to put up another issue for documentation just to make sure I've covered everything related to PRs and Code Reviews